mp/src/materialsystem/stdshaders/pbr_dx9.cpp

```diff
@@ -112,6 +112,10 @@ BEGIN_VS_SHADER(PBR, "PBR shader")
         if (!params[BUMPMAP]->IsDefined())
             params[BUMPMAP]->SetStringValue("dev/flat_normal");
 
+        // Set a good default mrao texture
+        if (!params[MRAOTEXTURE]->IsDefined())
+            params[MRAOTEXTURE]->SetStringValue("dev/pbr_mraotexture");
+
```

Comment:
Next time keep this to a separate branch, and just create branches off of branches if you require one branch's changes.

It becomes hell to review if you bundle too many things together.

Reply:
Not to mention this PR becomes an "all or nothing" situation, where changes could be held up from merging in because of something that could have been in another branch. Like, I approve of the PBR changes, but because you bundled it with the shader compile, you now have to address that.